taqueria-protocol/types-config-files.ts

```diff
@@ -293,6 +293,7 @@ export const transformConfigToConfigFileV2 = (config: Config): ConfigFileSetV2 =
 
 // FileV2 to Object
 export const transformConfigFileV2ToConfig = (configFileSetV2: ConfigFileSetV2): Config => {
+	debugger;
```

Comment:
debugger

Reply:
Gesh, I could have sworn that I searched for these in my editor before committing. Thanks for catching Jimmy.